packages/wrangler/src/create-worker-upload-form.ts

```diff
@@ -64,6 +64,7 @@ export interface WorkerMetadata {
 	bindings: WorkerMetadataBinding[];
 	keep_bindings?: WorkerMetadataBinding["type"][];
 	logpush?: boolean;
+	[key: string]: unknown;
```

Comment:
Could you add a comment here explaining that this is to cover unsafe metadata properties that cannot be known at transpilation time?